build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -4,12 +4,26 @@
   <PropertyGroup>
     <_SourceTopDir>..\..</_SourceTopDir>
     <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MandroidInternalDir>$(OutputPath)\lib\mandroid-internal</_MandroidInternalDir>
```

Comment:
I think we agreed on keeping remap tool in the monodroid for now and move it in the next step(s). 

I guess I can move it to the XA now then. It would probably make sense to use it on mono-symbolicate too.